### PR TITLE
[WICKET-6750] allow to abort currently executing AJAX request 

### PR DESCRIPTION
**Pull request by reiern70**
[WICKET-6750] allow to abort currently executing request for a given Ajax channel + example

### CODE REVIEW

File: wicket-core/src/main/java/org/apache/wicket/ajax/res/js/wicket-ajax-jquery.js [hunk 1]
```diff
@@ -547,13 +576,29 @@
...
-
 			var res = Wicket.channelManager.schedule(attrs.ch, Wicket.bind(function () {
 				this.doAjax(attrs);
 			}, this));
 			return res !== null ? res: true;
 		},
 
+		/**
+		 * Aborts current AJAX request, if any is running, for default channel.
+		 * WARNING! Mind that this does not implies and server immediately will know about
```
Comment (andruhon) [inline wicket-core/src/main/java/org/apache/wicket/ajax/res/js/wicket-ajax-jquery.js]: This does not imply that the server will immediately know about that? Is it what you're trying to tell?

I think this comment needs rewording.

Comment (andruhon) [reply wicket-core/src/main/java/org/apache/wicket/ajax/res/js/wicket-ajax-jquery.js]: I'd reword it somewhat like this:
```
Abort AJAX request if one in default channel is currently running.

WARNING! Mind that calling this method does not imply that the server will immediately know about the request being aborted.
Server side processing and page are likely to continue.
This method could be useful to cancel huge AJAX requests such as file uploads.
```

Comment (reiern70) [reply wicket-core/src/main/java/org/apache/wicket/ajax/res/js/wicket-ajax-jquery.js]: > This does not imply that the server will immediately know about that? Is it what you're trying to tell?
> 
> I think this comment needs rewording.

What I mean is "there is not reliable way to tell the server stop what I asked you to do". In case of upload stream will be broken and upload halted with a server side exception.

Comment (reiern70) [reply wicket-core/src/main/java/org/apache/wicket/ajax/res/js/wicket-ajax-jquery.js]: @andruhon thanks for your comment.

image

I have reworded it like above. Hope is is more clear now.
